install/src/main/java/org/jetbrains/bsp/bazel/install/cli/CliOptionsProvider.kt

```diff
@@ -187,6 +195,8 @@ class CliOptionsProvider(private val args: Array<String>) {
         private const val BAZEL_PATH_SHORT_OPT = "b"
         private const val DEBUGGER_ADDRESS_SHORT_OPT = "x"
         private const val JAVA_PATH_SHORT_OPT = "j"
+        private const val DIRECTORIES_SHORT_OPT = "r"
```

Comment:
u have to add these options in the `init {}` block as well + add in tests `CliOptionsProviderTest`

Reply:
Options in init{} block and test cases added